cogboard-app/src/main/kotlin/com/cognifide/cogboard/storage/ContentRepository.kt

```diff
@@ -4,9 +4,11 @@ import io.vertx.core.json.JsonObject
 import java.io.File
 
 class ContentRepository(private val path: String = "/data/content") {
+
     fun save(widgetId: String, content: JsonObject) {
-        File("$path/$widgetId.json")
-                .writeText(content.toString())
+        if (File(path).exists()) {
```

Comment:
this method is executed while JUnit test because it is a part of widget update process unfortunately while JUnit testing there is no Docker running so `$path` points nowhere

Reply:
IMO the proper fix would be in this case to make a different version of `ContentRepository` when tests are active, not conditioning it during normal run.

Reply:
This would cause even more `if` s